helm-charts/cs-k8s-protection-agent/templates/configmap.yaml

```diff
@@ -9,3 +9,14 @@ data:
   AGENT_DEBUG: {{ .Values.crowdstrikeConfig.enableDebug | default "false" | quote }}
   AGENT_ENV: {{ .Values.crowdstrikeConfig.env | quote }}
   AGENT_HELM_VERSION: {{ .Chart.Version | quote }}
+  {{- if .Values.proxyConfig }}
```

Comment:
This can be dropped once the proxy options are also added to the default values.yaml file

Reply:
If people are upgrading charts and using deploying using `--reuse-values` its probably still safer to have the if statements as this is not required configuration. Thoughts?

Reply:
Remove condition and set defaults